tests/e2e/utils/create-page.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import { cleanForSlug } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { apiWP } from './api';
+
+/**
+ * Check if a page exists from a title.
+ *
+ * @param {string} title
+ * @return {Promise<number>} Existing page ID.
+ */
+export async function pageExistsByTitle( title ) {
+	const slug = cleanForSlug( title );
+
+	return await apiWP()
+		.get( `pages?slug=${ slug }` )
+		.then( ( response ) => response.data[ 0 ]?.id );
+}
+
+/**
+ * Creates a WP page with content and title.
+ *
+ * @param {string} title
+ * @param {string} content
+ *
+ * @return {number} Created page ID.
+ */
+export async function createPage( title, content ) {
+	return await apiWP()
+		.post( 'pages', {
+			title,
+			content,
+			status: 'publish',
+		} )
+		.then( ( response ) => response.data.id );
+}
+
+/**
+ * Creates a classic cart page using shortcodes.
+ *
+ * @return {number} Created page ID.
+ */
+export async function createClassicCartPage() {
+	const title = 'Classic Cart';
+	const content = '[woocommerce_cart]';
+
+	if ( ! ( await pageExistsByTitle( title ) ) ) {
+		return await createPage( title, content );
+	}
```

Comment:
We're not returning the ID if the page exists.
```suggestion
	return (
		( await pageExistsByTitle( title ) ) ||
		( await createPage( title, content ) )
	);
```